diff --git a/node_modules/@vercel/node/dist/index.js b/node_modules/@vercel/node/dist/index.js
index 7f4aa9c..f108c55 100644
--- a/node_modules/@vercel/node/dist/index.js
+++ b/node_modules/@vercel/node/dist/index.js
@@ -70839,34 +70839,34 @@ async function forkDevServer(options) {
     });
     child.stderr?.on("data", console.error);
   } else {
-    let nodeOptions = process.env.NODE_OPTIONS || "";
-    if (!nodeOptions.includes("--no-warnings")) {
-      nodeOptions += " --no-warnings";
-    }
+    const execArgs = [];
     const tsNodePath = options.require_.resolve("ts-node");
     const esmLoader = (0, import_url.pathToFileURL)((0, import_path5.join)(tsNodePath, "..", "..", "esm.mjs"));
     const cjsLoader = (0, import_path5.join)(tsNodePath, "..", "..", "register", "index.js");
+    // prefer execArgv over NODE_OPTIONS to avoid issues with spaces in Windows paths
     if (options.maybeTranspile) {
       if (options.isTypeScript) {
-        nodeOptions = `--require ${cjsLoader} --loader ${esmLoader} ${nodeOptions || ""}`;
+        execArgs.push("--require", cjsLoader);
+        execArgs.push("--loader", String(esmLoader));
       } else {
         if (options.isEsm) {
         } else {
-          nodeOptions = `--require ${cjsLoader} ${nodeOptions || ""}`;
+          execArgs.push("--require", cjsLoader);
         }
       }
     }
+    // suppress warnings as before
+    execArgs.push("--no-warnings");
     const forkOptions = {
       cwd: options.workPath,
-      execArgv: [],
+      execArgv: execArgs,
       env: (0, import_build_utils6.cloneEnv)(process.env, options.meta.env, {
         VERCEL_DEV_ENTRYPOINT: options.entrypoint,
         VERCEL_DEV_CONFIG: JSON.stringify(options.config),
         VERCEL_DEV_BUILD_ENV: JSON.stringify(options.meta.buildEnv || {}),
         VERCEL_DEV_PUBLIC_DIR: options.publicDir || "",
         TS_NODE_TRANSPILE_ONLY: "1",
-        TS_NODE_COMPILER_OPTIONS: options.tsConfig?.compilerOptions ? JSON.stringify(options.tsConfig.compilerOptions) : void 0,
-        NODE_OPTIONS: nodeOptions
+        TS_NODE_COMPILER_OPTIONS: options.tsConfig?.compilerOptions ? JSON.stringify(options.tsConfig.compilerOptions) : void 0
       }),
       stdio: options.printLogs ? "pipe" : void 0
     };
